### DISCUSSION TITLE
MONAI Bundle tutorial training config file using PyTorch training loop

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1130] [Q&A]
Hi there!
In the MONAI Bundle tutorial for 3D spleen segmentation, the [training config json file](https://github.com/Project-MONAI/tutorials/blob/main/bundle/spleen_segmentation/configs/train.json) uses ignite-style of training, i,e, using SupervisedTrainer. Is there any example training config file using PyTorch training loop? Could you point me to that?

**Comment by KumoLiu** [comment]
Hi @hw-ju, since it's not easy to construct a PyTorch training loop into a config file. You can find many other examples under this folder. https://github.com/Project-MONAI/model-zoo/tree/dev/models
Thanks!

**Comment by Nic-Ma** [comment]
Hi @hw-ju ,

For flexible workflow, you can write a python script and combine it with components in the config file, refer to:
https://github.com/Project-MONAI/tutorials/tree/main/bundle/hybrid_programming

Thanks.
